### PR TITLE
Feature/material design changes

### PR DESCRIPTION
**Pull request by agduncan94**
This just includes some button changes from the discussion today, including changing a lot from icon/fab to regular mat-buttons.

### CODE REVIEW

File: src/app/container/container.component.html [hunk 1]
```diff
@@ -80,17 +80,17 @@ <h3 class="pull-right">
     <h3>
       <div class="pull-right my-entry-button-group">
         <div class="btn-group" role="group">
-          <button id="publishToolButton" mat-raised-button color="secondary" (click)="publish()" [(ngModel)]="published"
-            [disabled]="publishDisable()" btnCheckbox [matTooltipShowDelay]="1000" [matTooltipPosition]="'before'" matTooltip="Publish/unpublish the tool">
+          <button id="publishToolButton" mat-flat-button color="accent" (click)="publish()" [(ngModel)]="published"
+            [disabled]="publishDisable()" btnCheckbox matTooltip="Publish/unpublish the tool" class="mr-2">
```
Comment (denis-yuen) [inline src/app/container/container.component.html]: Minor thought. Is it possible to detect the button state and either say `Publish the tool to make it visible to the public` or `Unpublish the tool to remove it from the public` or similar (removing the slash and providing more detail)?

Same with workflows

File: src/app/mytools/my-tool/my-tool.component.html [hunk 2]
```diff
@@ -29,8 +29,8 @@
               <mat-progress-bar mode="indeterminate"></mat-progress-bar><br />
             </span>
             <div class="clearfix sidebar-header">
-                <button mat-mini-fab color="primary" [matTooltipShowDelay]="1000" [matTooltipPosition]="'after'" matTooltip="Refresh all of my tools" class="pull-right" (click)="refreshAllEntries()" [disabled]="refreshMessage">
-                  <mat-icon>refresh</mat-icon>
+                <button mat-flat-button color="primary" [matTooltipShowDelay]="1000" [matTooltipPosition]="'after'" matTooltip="Refresh all of my tools" class="pull-right" (click)="refreshAllEntries()" [disabled]="refreshMessage">
```
Comment (denis-yuen) [inline src/app/mytools/my-tool/my-tool.component.html]: "Refresh all of my tools and find new tools" perhaps

File: src/app/myworkflows/my-workflow/my-workflow.component.html [hunk 3]
```diff
@@ -29,8 +29,8 @@
           <mat-progress-bar mode="indeterminate"></mat-progress-bar><br />
         </span>
         <div class="clearfix sidebar-header">
-            <button mat-mini-fab color="primary" [matTooltipShowDelay]="1000" [matTooltipPosition]="'after'" matTooltip="Refresh all of my workflows" class="pull-right" (click)="refreshAllEntries()" [disabled]="refreshMessage">
-              <mat-icon>refresh</mat-icon>
+            <button mat-flat-button color="primary" [matTooltipShowDelay]="1000" [matTooltipPosition]="'after'" matTooltip="Refresh all of my workflows" class="pull-right" (click)="refreshAllEntries()" [disabled]="refreshMessage">
```
Comment (denis-yuen) [inline src/app/myworkflows/my-workflow/my-workflow.component.html]: Same deal, provide some indication that this is how we identify new repos

File: src/app/container/container.component.html (hunk 1 above)
Comment (agduncan94) [reply src/app/container/container.component.html]: It should be, I'll make the change.

**Pull request merged**